code/game/turfs/turf_changing.dm

```diff
@@ -2,6 +2,8 @@
 	var/base_turf = get_base_turf_by_area(src)
 	if(base_turf && type != base_turf)
 		. = ChangeTurf(base_turf)
+	else
+		. = src
```

Comment:
This could also be written as 
```
	if(base_turf && type != base_turf)
		src.ChangeTurf(base_turf)
	if(!(locate(/obj/structure/lattice) in src))
		new /obj/structure/lattice(src, material)
	return src
```
but this way is perfectly acceptable as well.